pages/linux/duperemove.md

```diff
@@ -0,0 +1,20 @@
+# duperemove
+
+> Finds duplicate extends and optionally schedule them for deduplication.
```

Comment:
We should clarify what an "extent" is and write it in a language that is easily understood.

Reply:
I tried to explain it, but I needed two more sentences for it. Hope this is ok ;)

Reply:
How about "Finds duplicate chunks within a file and optionally schedule them for deduplication." ?

Reply:
I'm wondering if chunk is better to understand than extent... If you think so, I'll change it.

Reply:
If an extent is defined as a part of a file, then I think check is easier to understand.

Reply:
But the man page and all other documentation about duperemove uses the term "extent" and I think we should use it too to avoid confusion. I think with the explanation it should be clear and I think that people, that do not have a basic knowledge about file systems won't use duperemove (and won't have btrfs/xfs).

Reply:
Ok, that's fine. We can keep it.